src/main/java/org/opentripplanner/model/calendar/impl/CalendarServiceDataFactoryImpl.java

```diff
@@ -44,43 +39,34 @@
     private final List<Agency> agencies;
     private final Map<FeedScopedId, List<ServiceCalendarDate>> calendarDatesByServiceId;
     private final Map<FeedScopedId, List<ServiceCalendar>> calendarsByServiceId;
-    private final Map<FeedScopedId, List<String>> tripAgencyIdsByServiceId;
     private final Set<FeedScopedId> serviceIds;
 
-    public static CalendarService createCalendarService(OtpTransitServiceBuilder transitDaoBuilder) {
-        return new CalendarServiceImpl(createCalendarServiceData(transitDaoBuilder));
-    }
-
-    public static CalendarServiceData createCalendarServiceData(OtpTransitServiceBuilder transitDaoBuilder) {
-        return new CalendarServiceDataFactoryImpl(transitDaoBuilder).createData();
-    }
+    public static CalendarServiceData createCalendarServiceData(
+            Collection<Agency> agencies,
```

Comment:
I do like this way of formatting function parameters, perhaps we should require it anytime the list does not fit on one line, or has more than 2-3 parameters). Something to add to the coding conventions.